tests/integration/src/setup.rs

```diff
@@ -107,6 +185,8 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
             (ALICE, INITIAL_AMOUNT),
             (BOB, INITIAL_AMOUNT),
             (CAT, INITIAL_AMOUNT),
+            #[cfg(feature = "shibuya")]
+            (alicia(), INITIAL_AMOUNT),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->